net.sf.eclipsecs.core/src/net/sf/eclipsecs/core/config/meta/CheckUtil.java

```diff
@@ -0,0 +1,75 @@
+package net.sf.eclipsecs.core.config.meta;
+
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.function.Function;
+import java.util.stream.Collectors;
+
+import com.puppycrawl.tools.checkstyle.PackageNamesLoader;
+import com.puppycrawl.tools.checkstyle.PackageObjectFactory;
+import com.puppycrawl.tools.checkstyle.api.AbstractCheck;
+import com.puppycrawl.tools.checkstyle.api.CheckstyleException;
+import com.puppycrawl.tools.checkstyle.checks.javadoc.AbstractJavadocCheck;
+import com.puppycrawl.tools.checkstyle.utils.JavadocUtil;
+import com.puppycrawl.tools.checkstyle.utils.TokenUtil;
+
+import net.sf.eclipsecs.core.CheckstylePlugin;
+
+public final class CheckUtil {
+    private CheckUtil() {
+    }
+
+    public static String getModifiableTokens(String checkName) {
+      final Object checkResult = getCheck(checkName);
+      String result = null;
+      if (AbstractJavadocCheck.class.isAssignableFrom(checkResult.getClass())) {
+          final AbstractJavadocCheck javadocCheck = (AbstractJavadocCheck) checkResult;
+          result = getTokens(JavadocUtil::getTokenName, subtractTokens(javadocCheck.getAcceptableJavadocTokens(),
+                  javadocCheck.getRequiredJavadocTokens()));
+
```

Comment:
remove this empty line and in next block

Reply:
Done